src/studio/src/designer/backend/Services/Implementation/RepositorySI.cs

```diff
@@ -162,7 +162,7 @@ public bool UpdateApplication(string org, string app, PlatformStorageModels.Appl
             }
             catch (Exception ex)
             {
-                _logger.LogError("Something went wrong when trying to update application metadata ", ex);
+                _logger.LogError(ex, "Something went wrong when trying to update application metadata.");
```

Comment:
Not related to your PR, but we should consider dropping the try/catch/log error/return false and instead have a void method which throws an exception instead. Thoughts?

Reply:
I do believe the catch might be doing more harm than good, but I didn't want to change functionality. I also think that exceptions here would be rare and if they do occur it is something we would want to fix. Not hide.

Reply:
I removed the try catch from a few of the methods. Baby steps.